scripts/travis/cleanup.js

```diff
@@ -1,22 +1,38 @@
 /*
- * Delete the artefact from the testing db
+ * Delete old artefacts from the testing db
  */
-
 const { UPLOAD_URL, TRAVIS_BUILD_NUMBER } = process.env;
+const BUILDS_TO_KEEP = 50;
+const END_BUILD_TO_DELETE = TRAVIS_BUILD_NUMBER - BUILDS_TO_KEEP;
+const END_KEY = `medic:medic:test-${END_BUILD_TO_DELETE}`;
+const MAX_BUILDS_TO_DELETE = 50;
 const PouchDB = require('pouchdb-core');
 PouchDB.plugin(require('pouchdb-adapter-http'));
 
 const db = new PouchDB(`${UPLOAD_URL}/_couch/builds_testing`);
-const docId = `medic:medic:test-${TRAVIS_BUILD_NUMBER}`;
 
 const get = () => {
-  console.log(`Getting "${docId}"...`);
-  return db.get(docId);
+  console.log(`Getting builds...`);
+  return db.allDocs({ endkey: END_KEY, limit: MAX_BUILDS_TO_DELETE });
```

Comment:
I did not think this worked!

If you've tested it then awesome, otherwise I think this has to be `descending: true` with `END_KEY` passed like `startkey: END_KEY`

Reply:
I tested it when this branch built and it cleaned up the oldest 50 builds as expected. I was a bit surprised too - I thought you needed a startkey...